connectors/byoc.py

```diff
@@ -207,7 +207,11 @@ def duration(self):
     def index_name(self):
         return self.doc_source["_source"]["connector"]["index_name"]
 
-    async def start(self, trigger_method=JobTriggerMethod.SCHEDULED, filtering=None):
+    async def start(self, connector, trigger_method=JobTriggerMethod.SCHEDULED, filtering=None):
+        # Okay that's temporary here. When we separate scheduling from execution , this code will go away
```

Comment:
I would add `XXX` or `TODO` as a prefix to this comment

Reply:
++ for "# TODO:", which is caught by nearly every IDE and shows up in the corresponding TODOs tab.